### PR TITLE
Updating copyright year 2022

### PR DESCRIPTION
**Pull request by bbpgithubaudit**

### CODE REVIEW

File: CMake/CMakeISPCCompiler.cmake.in [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2016 - 2021 Blue Brain Project/EPFL
+# Copyright (c) 2016 - 2021-2022 Blue Brain Project/EPFL
```
Comment (olupton) [inline CMake/CMakeISPCCompiler.cmake.in:2]: This doesn't seem quite right.

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2016 - 2022 Blue Brain Project/EPFL
+# Copyright (c) 2016 - 2022-2022 Blue Brain Project/EPFL
```
Comment (olupton) [inline CMakeLists.txt:2]: This doesn't seem quite right.

**Review by olupton — CHANGES_REQUESTED** [review]
Looks like the bot has got confused here.

File: CMakeLists.txt (hunk 2 above)
Comment (alex4200) [reply CMakeLists.txt:2]: Please close this PR. 

I have created a new PR: https://github.com/BlueBrain/CoreNeuron/pull/819

**Pull request closed without merging**